dis_snek/models/discord_objects/user.py

```diff
@@ -423,6 +431,28 @@ def has_role(self, *roles: Union[Snowflake_Type, Role]) -> bool:
                 return False
         return True
 
+    async def timeout(self, communication_disabled_until: Union[Timestamp, datetime] = MISSING, reason: str = MISSING):
+        """
+        Disable a members communication for a given time.
+
+        Args:
+            communication_disabled_until: The time until the user can communicate again (default 1 minute)
+        """
+        if communication_disabled_until is MISSING:
+            communication_disabled_until = Timestamp.now() + timedelta(minutes=1)
```

Comment:
This is undocumented and inherantly unpredictable behaviour. This method should not have a default duration for timing someone out, it should be explictly set.